concent_api/verifier/utils.py

```diff
@@ -313,7 +315,7 @@ def render_image(frame_number, output_format, scene_file, subtask_id, verificati
         )
 
 
-def unpack_archives(file_paths, subtask_id):
+def unpack_archives(file_paths: Iterable[str], subtask_id: str) -> None:
```

Comment:
How about changing this into `unpack_archives(subtask_id: str, *args)` where `args` will be list of paths ? What do you guys think ?

Reply:
Hmm, I think that it is good to know exactly what parameters function gets, but it would be also good solution. I'm leaving this decision to you guys @rwrzesien @dybi :)

Reply:
It is just a proposal, I have done it in the same way in function that validates message signature. But your comment has also a point. So maybe we shouldn't do it yet.

Reply:
If we know exactly the number and type of arguments, I would be for leaving as it is ;)